test/e2e/test/logstash/steps.go

```diff
@@ -145,5 +147,15 @@ func (b Builder) DeletionTestSteps(k *test.K8sClient) test.StepList {
 				return k.CheckPodCount(0, b.ListOptions()...)
 			}),
 		},
+		{
+			Name: "Cleanup any persistent volumes belonging to Logstash",
+			Test: test.Eventually(func() error {
+				if err := k.Client.DeleteAllOf(context.Background(), &corev1.PersistentVolumeClaim{},
+					client.MatchingLabels{lslabels.NameLabelName: b.Logstash.Name}); err != nil && !apierrors.IsNotFound(err) {
```

Comment:
I think you need to specify a namespace otherwise the request will be refused.

Reply:
Thanks @thbkrkr I did not think of that.